### PR TITLE
Homepage updates

### PR DESCRIPTION
**Pull request by ikesau**
Resolves (most of) [#3290](https://github.com/owid/owid-grapher/issues/3290)

### CODE REVIEW

**Comment by marcelgerber** [comment]
Hi! I don't think adding a method to find `grapherId`-based explorers is a natural contender for #3350.
We currently have this weird conundrum that most of our explorer code is based off file system reads, but some of the "analytics"-like requests are served from the DB instead (which should have the same content -- eventually).

Here's a query you can use to find the number of non-grapherId views:
```sql
SELECT COUNT(*)
FROM explorers,
     json_table(config, "$.blocks[*]"
                COLUMNS (
                    type TEXT PATH "$.type",
                    NESTED PATH "$.block[*]"
                        COLUMNS (grapherId INT PATH "$.grapherId")
                    )
     ) t1
WHERE isPublished = 1
  AND type = "graphers"
  AND grapherId IS NULL
```

Maybe @danyx23 has a good idea how best to incorporate this query into the baking process, or where to put it?

**Comment by danyx23** [comment]
Nice query, @marcelgerber :). I think putting this into the db folder, maybe in a new file, and calling it during baking sounds like a pretty good solution.

File: db/model/Gdoc/GdocHomepage.ts [hunk 1]
```diff
@@ -63,8 +63,12 @@ export class GdocHomepage
     _loadSubclassAttachments = async (
         knex: db.KnexReadonlyTransaction
     ): Promise<void> => {
+        const grapherCount = await db.getTotalNumberOfCharts(knex)
+        const nonGrapherExplorerViewCount =
+            await db.getNonGrapherExplorerViewCount(knex)
```
Comment (marcelgerber) [inline db/model/Gdoc/GdocHomepage.ts:68]: Can maybe optimize this a tiny bit by using `Promise.all` to parallelize these two.

File: site/search/searchClient.ts [hunk 2]
```diff
@@ -52,4 +52,4 @@ export const logSiteSearchClickToAlgoliaInsights = (
 }
 
 export const DEFAULT_SEARCH_PLACEHOLDER =
-    "Try “Life expectancy”, “Poverty Nigeria Vietnam”, “CO2 France”…"
+    "Try “Life expectancy”, “Economic growth”, “Homicide rate”, “Biodiversity”…"
```
Comment (marcelgerber) [inline site/search/searchClient.ts:55]: Ay you don't want to change this one

**Review by marcelgerber — APPROVED** [review]
Very nice!

**Pull request merged**